src/main/java/school/lemon/changerequest/java/introduction/hw1/Task8.java

```diff
@@ -2,6 +2,10 @@
 
 public class Task8 {
     public static int calculateSum(int N) {
-        return 0;
+        int sum = 0;
+        for (int i = 0; i <= N; i++) {
+            sum += Math.pow(N + i, 2);
```

Comment:
1. I'd prefer use multiplication instead of pow function.
2. You can avoid `N +` just initialize `i = N` instead of `0`.